src/palette/elements/Input/Input.tsx

```diff
@@ -90,6 +91,7 @@ export const Input = React.forwardRef<TextInput, InputProps>(
       multiline,
       maxLength,
       showLimit,
+      shouldAddListenerForClear = true,
```

Comment:
Why not default to `false` since we just need the clear listener in one place in the app (Search Screen)?

```suggestion
      addClearListener = false,
```

Reply:
I didn't know we needed it only in one place. It definitely should default to false. 👍